code/datums/atmos/_atmos.dm

```diff
@@ -0,0 +1,50 @@
+/datum/atmos
```

Comment:
might be better to rename to `/datum/atmosphere` instead since atmos is usually read as atmospherics, this had me confused for a bit before I figured out the intent

Reply:
fair point